### PR TITLE
 Add version as global from package.json

### PR DESCRIPTION
**Pull request by mertkahyaoglu**
One less file change for releases!

### CODE REVIEW

File: src/containers/Header.js [hunk 1]
```diff
@@ -1,9 +1,6 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
-import { connect } from 'react-redux';
-import { bindActionCreators } from 'redux';
```
Comment (ashmaroli) [inline src/containers/Header.js]: Technically, these changes don't belong to this PR and hence would be best pushed to `master` directly..

**Comment by ashmaroli** [comment]
> it is easy to review these changes and saves you lots of time.

For me, it was the opposite. When I saw the deleted lines in `Header.js`, I had to further investigate why those two lines were deleted when they had nothing to do with emitting the version globally..

**Comment by ashmaroli** [comment]
Since we don't have strict guidelines in place, I won't let those extra changes be a blocker.
I shall approve this once I figure out how best to get the Coveralls passing again without going astray too far..

File: package.json [hunk 2]
```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "start-message": "babel-node tools/startMessage.js",
     "prestart": "npm-run-all --parallel start-message remove-dist",
-    "start": "npm-run-all --parallel start-api test:watch open:src lint:watch",
```
[the remark this answers is not in the document]
Comment (ashmaroli) [reply package.json]: Resolved by adding a new task `start:tested` to `master` directly.

**Comment by ashmaroli** [comment]
@mertkahyaoglu If you won't be adding any more changes to this branch. I shall merge it in.
Is this good to go..?

**Comment by mertkahyaoglu** [comment]
Yeap 👍

**Comment by ashmaroli** [comment]
I just ran the coverage task locally.. Looks like there's still a `0.01%` decrease..
